Makefile

```
@@ -45,14 +45,35 @@ test: ## Run tests for operator and agent.
 	$(MAKE) -C operator fmt
 	$(MAKE) -C agent fmt
 
 .PHONY: license-fmt
 license-fmt: ## Run license header formatting for all code.
-	python3 scripts/format_license.py --root-dir . --license-file LICENSE
+	$(MAKE) -C operator license-fmt
+	$(license_files) | xargs $(ADDLICENSE) -c "$(LICENSE_HOLDER)" -f $(LICENSE_TEMPLATE)
+	$(MAKE) -C agent license-fmt
+
+.PHONY: license-header-check
+license-header-check: ## Check license headers for all code.
```

checkmake (0.3.2)

[warning] Target body for "license-header-check" exceeds allowed length of 5 lines (10).

(maxbodylength)